DependencyInjection/HttplugExtension.php

```diff
@@ -56,6 +61,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_journal_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.plugin.history');
```

Comment:
We cant do a search/replace. People should not use "our" namespace when defining services so they would create a service like `my_history_plugin`. So it is impossible to do a search because we dont know what to search for... 

If someone uses our service, then the history will be logged twice. The readme says it is added automatically. We should not bother. 

What we should care about is that what if they are not happy with our formatter? But I think that can be solved by #17.